src/styles.css

```diff
@@ -0,0 +1,1174 @@
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+}
+
+.blur {
+  filter: blur(5px);
+}
+
+@media screen and (min-width: 900px) {
+  .body {
+    margin: 0;
+    background-color: #e5e5e5;
+  }
+
+  .header-container {
+    display: none;
+  }
+
+  .web-nav {
+    display: flex;
+  }
+
+  .Mylogo {
+    text-decoration: none;
+    color: #6070ff;
+  }
+
+  .modal-example {
+    width: 60%;
+    padding: 2%;
+  }
+
+  .modal-example.show {
+    right: 0;
+    max-width: 100%;
+    bottom: 0;
+    max-height: 608px;
+    transform: translate(-10%, 0%);
+  }
+
+  .modal__body {
+    overflow-y: auto;
+  }
+
+  .modal-img {
+    width: 90%;
+  }
+
+  .desc {
+    width: 50%;
+  }
+
+  nav {
+    display: flex;
+    height: 80px;
+    width: 97%;
+    background: #fff;
+    align-items: center;
+    justify-content: space-between;
+    padding: 10px;
+    flex-wrap: wrap;
+  }
+
+  nav .logo {
+    color: #6070ff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    transition: all 1s;
+  }
+
+  .logo:hover {
+    transform: rotate(360deg);
+  }
+
+  nav ul {
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+  }
+
+  nav ul li {
+    margin: 0 5px;
+  }
+
+  nav ul li a {
+    color: #344563;
+    text-decoration: none;
+    padding: 8px 15px;
+    border-radius: 5px;
+    letter-spacing: 1px;
+    transition: all 0.3s ease;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  nav ul li a:hover {
+    color: #fff;
+    background: #344563;
+  }
+
+  .content {
+    background-image: url(../assets/Header-bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 99%;
+    height: 100vh;
+    border-radius: 0 0 0 100px;
+    border-bottom: 1px solid #cfd8dc;
+    background-color: #fff;
+  }
+
+  .content p {
+    width: 63%;
+    margin-left: 20%;
+    margin-top: -5%;
+    line-height: 28px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .content h3 {
+    margin-left: 20%;
+    margin-top: -5%;
+  }
+
+  .content ul {
+    margin-left: 20%;
+    margin-top: 1%;
+  }
+
+  .intro {
+    width: 400px;
+    font-size: 48px;
+    line-height: 60px;
+    margin-left: 22%;
+    animation-name: introin;
+    animation-duration: 4s;
+  }
+
+  @keyframes introin {
+    from {
+      margin-left: 100%;
+    }
+
+    to {
+      margin-left: 22%;
+    }
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: row;
+    margin: 5%;
+    width: 100%;
+    height: 60%;
+  }
+
+  .img-web {
+    display: flex;
+    width: 40%;
+  }
+
+  .project-description {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .project-details {
+    align-items: center;
+  }
+
+  .project-desc {
+    display: flex;
+  }
+
+  .role {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
- hey, let's adjust the spaces between these items to be close to the Figma design:


*Figma design*
![Screenshot from 2023-05-02 00-14-58](https://user-images.githubusercontent.com/59397159/235547451-f24b020a-3d76-4ec2-b1f4-2e7500374077.png)


*your  design*
![Screenshot from 2023-05-02 00-15-44](https://user-images.githubusercontent.com/59397159/235547542-241556e1-81dd-4353-b3c5-4b1cc460b9f5.png)